cmd/kube-proxy/app/options/options.go

```diff
@@ -57,7 +57,6 @@ func NewProxyConfig() *ProxyServerConfig {
 			ResourceContainer:              "/kube-proxy",
 			IPTablesSyncPeriod:             unversioned.Duration{30 * time.Second},
 			UDPIdleTimeout:                 unversioned.Duration{250 * time.Millisecond},
-			Mode:                           componentconfig.ProxyModeUserspace,
```

Comment:
Can you set explicitly to componentconfig.ProxyModeIPTables. That will show the correct defaulting in the --help output